Formula/ocp.rb

```diff
@@ -21,30 +21,41 @@ class Ocp < Formula
   depends_on "pkg-config" => :build
   depends_on "xa" => :build
   depends_on "flac"
+  depends_on "freetype"
   depends_on "jpeg-turbo"
   depends_on "libpng"
   depends_on "libvorbis"
   depends_on "mad"
+  depends_on "sdl2"
```

Comment:
Does this also need `sdl2_mixer`? I think that's the SDL sound library. Though I think `sdl2` on its own has some support.

Reply:
The `sdl2_mixer` description seems to indicate that it's a multi-channel audio mixer library, so I think we want just `sdl2`. Though of course, I'm only saying this because it works on my system 😅

Reply:
> it works on my system

Works for me!

Reply:
I can now confirm that we don't require it – `ocp` only requires `SDL.h` and `SDL_audio.h`, which `sdl2` provides.